crates/autopilot/src/infra/persistence/dto/fee_policy.rs

```diff
@@ -0,0 +1,47 @@
+use {
+    crate::domain::{fee::Policy, AuctionId, OrderUid},
+    database::{
+        byte_array::ByteArray,
+        fee_policies::{FeePolicyKindRow, FeePolicyRow},
+    },
+};
+
+impl From<FeePolicyRow> for Policy {
```

Comment:
Can the `FeePolicyRow` struct be lifted up here or would this cause other weird dependencies? I find it weird to have a `dto::fee_policy` model that is not really defining any dto.

Also, I'd prefer this to be available via `database::FeePolicy` or `dto::FeePolicy` rather than `FeePolicyRow` (while this naming is consistent with the database crate, it doesn't really fit into the new DDD types, maybe we can re-export it somehow?)

Reply:
Moved together with the `database::fee_policies`.